src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+import type {ElementRef} from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import checkboxStyles from './checkbox.scss';
+
+type Props = {
+    checked: boolean,
```

Comment:
@mmsbrggr not really true, and I think in some cases it would be convenient ... https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#Value
it's also about having a consistent interface, e.g. radios already have values